aspnetcore/tutorials/signalr.md

```diff
@@ -212,52 +213,53 @@ The preceding highlighted code adds SignalR to the ASP.NET Core dependency injec
 
 ## Add SignalR client code
 
-1. Replace the content in `Pages/Index.cshtml` with the following code:
+Replace the content in `Pages/Index.cshtml` with the following code:
 
-   [!code-cshtml[Index](~/tutorials/signalr/samples/SignalRChat/Pages/Index.cshtml)]
+[!code-cshtml[Index](~/tutorials/signalr/samples/SignalRChat/Pages/Index.cshtml)]
 
-   The preceding markup:
+The preceding markup:
 
-   * Creates text boxes and a submit button.
-   * Creates a list with `id="messagesList"` for displaying messages that are received from the SignalR hub.
-   * Includes script references to SignalR and the `chat.js` app code is created in the next step.
+* Creates text boxes and a submit button.
+* Creates a list with `id="messagesList"` for displaying messages that are received from the SignalR hub.
+* Includes script references to SignalR and the `chat.js` app code is created in the next step.
 
-1. In the `wwwroot/js` folder, create a `chat.js` file with the following code:
+In the `wwwroot/js` folder, create a `chat.js` file with the following code:
 
-   [!code-javascript[chat](~/tutorials/signalr/samples/SignalRChat/wwwroot/js/chat.js)]
+[!code-javascript[chat](~/tutorials/signalr/samples/SignalRChat/wwwroot/js/chat.js)]
 
-   The preceding JavaScript:
+The preceding JavaScript:
 
-   * Creates and starts a connection.
-   * Adds to the submit button a handler that sends messages to the hub.
-   * Adds to the connection object a handler that receives messages from the hub and adds them to the list.
+* Creates and starts a connection.
+* Adds to the submit button a handler that sends messages to the hub.
+* Adds to the connection object a handler that receives messages from the hub and adds them to the list.
 
 ## Run the app
 
 # [Visual Studio](#tab/visual-studio)
 
-1. Press CTRL+F5 to run the app without debugging.
+Press CTRL+F5 to run the app without debugging.
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-1. In the integrated terminal, run the following command:
+In the integrated terminal, run the following command:
 
-  ```dotnetcli
-  dotnet watch run --project SignalRChat.csproj
-  ```
+```dotnetcli
+dotnet watch run --project SignalRChat.csproj
+```
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
 1. From the menu, select **Run > Start Without Debugging**.
```

Comment:
Looks like you missed a numbered step

Reply:
Hmm, sorry I missed that.  I remember doing a search to make sure I didn't miss one, and yet ...I still missed it.  Thanks.

Reply:
..interesting.  I actually did have that numbered step fixed already before in commit [28590](https://github.com/dotnet/AspNetCore.Docs/pull/28590/commits/5118771af2c07cf6315b9739bb76ca04c1cbfe70) before I put in the review request.  I was was double checking all that, so I felt ashamed that I might have missed it anyway.  I think what happened is you hit it right away (thanks) and the build was generating, (which I should have waited to verify it was fine in the build before asking for a review, more shame, haha).